pyqtgraph/debug.py

```diff
@@ -535,7 +535,7 @@ def mark(self, msg=None):
             pass
     _disabledProfiler = DisabledProfiler()
         
-    def __new__(cls, msg=None, disabled='env', delayed=True):
+    def __new__(cls, msg=None, disabled='env', delayed=True, gt: float = 0):
```

Comment:
probably should make the default value be `0.` or `0.0` if it's going to be type-annotated to a float.

Reply:
good idea, are we cool with introducing type annots now?